scripts/validate.sh

```diff
@@ -58,7 +58,7 @@ find . -type f -name $kustomize_config -print0 | while IFS= read -r -d $'\0' fil
   do
     echo "INFO - Validating kustomization ${file/%$kustomize_config}"
     kustomize build "${file/%$kustomize_config}" $kustomize_flags | \
-      kubeval --ignore-missing-schemas --strict --additional-schema-locations=file:///tmp/flux-crd-schemas
+      kubeconform -ignore-missing-schemas -strict -schema-location=file:///tmp/flux-crd-schemas
```

Comment:
Does this validates the Kubernetes native kinds too? Don't we need to add another `-schema-location` to point to the Kubernetes schemas? Also please add `-summary` to both commands, so we can see what it checks for.